content/en/docs/tutorials/stateful-application/cassandra.md

```diff
@@ -103,147 +109,143 @@ The StatefulSet manifest, included below, creates a Cassandra ring that consists
-      # and an empty file will abort the edit. If an error occurs while saving this file will be
-      # reopened with the relevant failures.
-      #
-      apiVersion: apps/v1 # for versions before 1.9.0 use apps/v1beta2
-      kind: StatefulSet
-      metadata:
-        creationTimestamp: 2016-08-13T18:40:58Z
-        generation: 1
-        labels:
-          app: cassandra
-        name: cassandra
-        namespace: default
-        resourceVersion: "323"
-        selfLink: /apis/apps/v1/namespaces/default/statefulsets/cassandra
-        uid: 7a219483-6185-11e6-a910-42010a8a0fc0
-      spec:
-        replicas: 3
-      ```
-
-1. Change the number of replicas to 4, and then save the manifest. 
+    ```shell
+    kubectl edit statefulset cassandra
+    ```
+
+    This command opens an editor in your terminal. The line you need to change is the `replicas` field. The following sample is an excerpt of the StatefulSet file:
+
+    ```yaml
+    # Please edit the object below. Lines beginning with a '#' will be ignored,
+    # and an empty file will abort the edit. If an error occurs while saving this file will be
+    # reopened with the relevant failures.
+    #
+    apiVersion: apps/v1 # for versions before 1.9.0 use apps/v1beta2
+    kind: StatefulSet
+    metadata:
+      creationTimestamp: 2016-08-13T18:40:58Z
+      generation: 1
+      labels:
+      app: cassandra
+      name: cassandra
+      namespace: default
+      resourceVersion: "323"
+      selfLink: /apis/apps/v1/namespaces/default/statefulsets/cassandra
+      uid: 7a219483-6185-11e6-a910-42010a8a0fc0
+    spec:
+      replicas: 3
+    ```
+
+2. Change the number of replicas to 4, and then save the manifest. 
 
    The StatefulSet now contains 4 Pods.
 
 1. Get the Cassandra StatefulSet to verify:
 
-      ```shell
-      kubectl get statefulset cassandra
-      ```
+    ```shell
+    kubectl get statefulset cassandra
+    ```
 
-      The response should be
+    The response should be
 
-      ```
-      NAME        DESIRED   CURRENT   AGE
-      cassandra   4         4         36m
-      ```
+    ```
+    NAME        DESIRED   CURRENT   AGE
+    cassandra   4         4         36m
+    ```
       
 {{% /capture %}}
 
 {{% capture cleanup %}}
 Deleting or scaling a StatefulSet down does not delete the volumes associated with the StatefulSet. This ensures safety first: your data is more valuable than an auto purge of all related StatefulSet resources. 
 
 {{< warning >}}
-**Warning:** Depending on the storage class and reclaim policy, deleting the Persistent Volume Claims may cause the associated volumes to also be deleted. Never assume you’ll be able to access data if its volume claims are deleted.
+**Warning:** Depending on the storage class and reclaim policy, deleting the PersistentVolumeClaims may cause the associated volumes to also be deleted. Never assume you’ll be able to access data if its volume claims are deleted.
```

Comment:
1. Same question about admonition text and #9482. 

2. Thinking about [inline code formatting](https://kubernetes.io/docs/home/contribute/style-guide/#inline-code-formatting): should `PersistentVolumeClaims` be code styled because it's an object?

Reply:
@zacharysarah Both admonitions in this text would indeed be covered by #9482. I'm 100% happy to be wrong, but I don't think I've seen an admonition in the k8s docs that doesn't, which was the original inspiration behind that PR. If you think that we should continue to require this to be made, explicit, though, I'm happy to discontinue that effort.